cns/dockerclient/dockerclient.go

```diff
@@ -55,6 +55,8 @@ func (dockerClient *DockerClient) NetworkExists(networkName string) error {
 		return err
 	}
 
+	defer res.Body.Close()
```

Comment:
If you need to Close sockets multiple times, you should give those _res_ different names. Even though they're pointers & should work fine, it will still cause confusion.